### PR TITLE
Add Missing Kopia Wrappers in Kanister

### PR DESCRIPTION
**Pull request by r4rajat**
## Change Overview
Add Missing Kopia Wrappers in Kanister
- Add wrapper for RepositoryStatusCommand 
- Add wrapper for GeneralCommand
- Update Wrapper SnapshotCreate 
## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: pkg/kopia/command/parse_command_output.go [hunk 1]
```diff
@@ -153,7 +153,7 @@ func sumSnapshotSizes(snapList []*snapshot.Manifest) (sum int64) {
 	return sum
 }
 
-func parseSnapshotManifestList(output string) ([]*snapshot.Manifest, error) {
+func ParseSnapshotManifestList(output string) ([]*snapshot.Manifest, error) {
```
Comment (akankshakumari393) [inline pkg/kopia/command/parse_command_output.go:156]: Are we calling this function from any other package? If not, then we should not export it.

**Review by ankitjain235 — COMMENTED** [review]
Changes LGTM. Let's add tests for the utilities added.

File: pkg/kopia/command/parse_command_output.go (hunk 1 above)
Comment (r4rajat) [reply pkg/kopia/command/parse_command_output.go]: Thanks ! made the changes

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -77,3 +79,29 @@ func GetEnvAsStringOrDefault(envKey string, def string) string {
 
 	return def
 }
+
+// DurationToString formats the given duration into a short format which eludes trailing zero units in the string.
```
Comment (akankshakumari393) [inline pkg/utils/utils.go:83]:
```suggestion
// DurationToString formats the given duration into a short format which exludes trailing zero units in the string.
```

**Review by akankshakumari393 — COMMENTED** [review]
LGTM

File: pkg/kopia/command/common.go [hunk 3]
```diff
@@ -26,6 +26,14 @@ type CommandArgs struct {
 	LogDirectory   string
 }
 
+type KopiaCommandParams struct {
```
Comment (ankitjain235) [inline pkg/kopia/command/common.go:29]: We can just use the new struct `GeneralCommandArgs` now instead of these two separate structs.

**Review by ankitjain235 — COMMENTED** [review]
Minor comment, looks good otherwise.

File: pkg/kopia/command/parse_command_output.go (hunk 1 above)
Comment (ankitjain235) [reply pkg/kopia/command/parse_command_output.go]: @r4rajat We may use it in other packages soon. We can keep it exported.

File: pkg/kopia/command/repository.go [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+type RepositoryStatusCommandArgs struct {
+	*CommandArgs
+}
+
+func RepositoryStatusCommand(cmdArgs RepositoryStatusCommandArgs) []string {
```
Comment (ankitjain235) [inline pkg/kopia/command/repository.go:21]: Let's add comments to all exported functions as well.

File: pkg/kopia/command/parse_command_output.go (hunk 1 above)
Comment (r4rajat) [reply pkg/kopia/command/parse_command_output.go]: Okay Will do that

File: pkg/kopia/command/common.go (hunk 3 above)
Comment (r4rajat) [reply pkg/kopia/command/common.go:29]: Okay will make changes !

File: pkg/kopia/command/repository.go (hunk 4 above)
Comment (r4rajat) [reply pkg/kopia/command/repository.go:21]: Sure !

File: pkg/kopia/command/common.go (hunk 3 above)
Comment (r4rajat) [reply pkg/kopia/command/common.go]: Made the changes !

**Review by ankitjain235 — APPROVED** [review]
LGTM

**Pull request merged**